stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -200,7 +201,7 @@ swift_reflection_addReflectionInfo(SwiftReflectionContextRef ContextRef,
       || Info.capture.offset != 0
       || Info.type_references.offset != 0
       || Info.reflection_strings.offset != 0) {
-    fprintf(stderr, "reserved field in swift_reflection_info_t is not zero\n");
+    std::cout << "reserved field in swift_reflection_info_t is not zero\n";
```

Comment:
This one should definitely be `cerr`.

Reply:
This is more of an assertion failure than an error, but better handling wouldn't be a bad thing.